core/src/main/java/io/questdb/TelemetrySystemEvent.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb;
+
+public final class TelemetrySystemEvent {
+    public static final short APPLIED_TXN = 103;
+    public static final short APPLY_STRUCTURE_CHANGE = 104;
+    public static final short APPLY_TXN_DATA = 105;
+    public static final short APPLY_TXN_SQL = 106;
+    public static final short ILP_RESERVE_WRITER = 102;
+    public static final short SYSTEM_DOWN = 101;
+    public static final short SYSTEM_UP = 100;
+}
```

Comment:
Can we track WAL apply failure events too?

Reply:
we can add probably `WAL_APPLY_SUSPEND` and `WAL_APPLY_RESUME` events.
for suspend we can log error code, error message.
this would be a separate table probably.

Reply:
Added in https://github.com/questdb/questdb/pull/2930/commits/0afb9f9bdc03a9fe6f54fa07aa1830c57a3ae401